deploy/vagrant/conf/ec2.yml.template

```diff
@@ -11,7 +11,7 @@ Key_Path: "~/.ssh/aws-east.pem"
 # When using spark, we recommend "r3.large", compile spark needs 2G memory at least
 #
 # Some Instance Type must need PV/HVM(refer to AMI below)
-Instance_Type: "t2.small"
+Instance_Type: "r3.large"
```

Comment:
hmm, then shall we call this out in the tutorial to warn the users about the cost per hour?